src/types/Operation.ts

```diff
@@ -70,4 +70,15 @@ export class Operation {
             amount: new Amount(Number(amount)).getObject()
         }
     }
+
+    static createNew(identifierIndex: number, account: string, amount: number | string, type: OperationTypes = OperationTypes.Transfer):IOperation {
```

Comment:
Any reason for the New part in the name? It is exactly the same as the above function but without the status right (and that already has a default)?

Reply:
The rosetta CLI would throw an error when there is a operation that has a status field in it while creating a new transaction

Reply:
What about removing the default value then?

Reply:
I don't exactly recall the problem but that was causing issues for the check:data test as it was expecting them

Reply:
Okay, don't really like it, but lets leave it in then